README.md

```diff
@@ -116,6 +117,13 @@ repositories:
 ```
 _Note: The `summary` and `introduction` fields are to be rendered as markdown, allowing for bullet points, links and other simple formatting._
 
+#### Project Icons
+
+Each project should have an icon associated to it to be displayed on the home page. To pick an icon, you could either:
+1. Pick an icon from [materialdesignicons](https://materialdesignicons.com) and include its name in the `icon` field of the yaml file.
+2. Include an icon in `src/images/projectIcons`. The icon should have the same name as its configuration file
+and have the image extension set as `.svg`.
```

Comment:
In this case, should I not include `icon:` in the yaml?

Reply:
Correct.